src/BrazeEndOfArticleComponent.ts

```diff
@@ -1,29 +1,46 @@
 import React from 'react';
 
+// Common Epic imports
+import {
+    buildBrazeMessageComponent,
+    ComponentMapping,
+    HasConsolidatedTrackClick,
+} from './buildBrazeMessageComponent';
+import type { BrazeClickHandler, SubmitComponentEvent } from './utils/tracking';
+
+// Why are we mapping this component?
+// Isn't it the blueprint on top of which other newsletter epics get built?
 import {
     COMPONENT_NAME as NEWSLETTER_EPIC_NAME,
     NewsletterEpic,
     NewsletterSubscribeCallback,
 } from './NewsletterEpic';
 
+// Generic Epic Templates
+import { COMPONENT_NAME as EPIC_NAME, Epic } from './Epic';
+import {
+    COMPONENT_NAME as EPIC_WITH_HEADER_IMAGE_NAME,
+    EpicWithSpecialHeader,
+} from './EpicWithSpecialHeader';
+
+// Old name Newsletter Epics
 import { COMPONENT_NAME as US_NEWSLETTER_EPIC_NAME, USNewsletterEpic } from './USNewsletterEpic';
 import { COMPONENT_NAME as AU_NEWSLETTER_EPIC_NAME, AUNewsletterEpic } from './AUNewsletterEpic';
 import { COMPONENT_NAME as UK_NEWSLETTER_EPIC_NAME, UKNewsletterEpic } from './UKNewsletterEpic';
 import {
     COMPONENT_NAME as DTE_NEWSLETTER_EPIC_NAME,
     DownToEarthNewsletterEpic,
 } from './DownToEarthNewsletterEpic';
-import { COMPONENT_NAME as EPIC_NAME, Epic } from './Epic';
+
+// New name Newsletter Epics
 import {
-    COMPONENT_NAME as EPIC_WITH_HEADER_IMAGE_NAME,
-    EpicWithSpecialHeader,
-} from './EpicWithSpecialHeader';
+    COMPONENT_NAME as EPICNEWSLETTER_AU_AFTERNOONUPDATE_NAME,
+    EpicNewsletter_AU_AfternoonUpdate,
+} from './EpicNewsletter_AU_AfternoonUpdate';
```

Comment:
No big deal either way, but I wonder if it'd be nice to be consistent with naming? Could this be `AusAfternoonUpdateNewsletterEpic` for example (and `TheGuideNewsletterEpic` for the other one)? (to have a common suffix with the existing components)

Reply:
My personal preference is for semantic naming - major_minor_whatever - which I think should be EpicNewsletter_REGION-if-applicable_newsletter-name ... if that makes sense?

It's a moot thought though, given that we'll shortly be moving to a more self-serve template so we don't have to build these bespoke templates anymore